### PR TITLE
ci: enable orka

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Enable MacOSX ephemeral workers

## Why is it important?

Validate we can use the MacOSX ephemeral workers. 

## Issues

Already verified in:

- https://github.com/elastic/beats/pull/29372
- https://github.com/elastic/beats/pull/29371
- https://github.com/elastic/beats/pull/29345

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -680,6 +680,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
       dockerLogin(secret: "${DOCKER_ELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
       dockerLogin(secret: "${DOCKERHUB_SECRET}", registry: 'docker.io')
     }
+    dumpVariables()
```
Comment (v1v) [inline Jenkinsfile:683]:
```suggestion
    if (nodeOS().contains('darwin')) {
      sh 'env | sort'
    }
```

File: Jenkinsfile [hunk 2]
```diff
@@ -680,6 +680,9 @@ def withBeatsEnv(Map args = [:], Closure body) {
       dockerLogin(secret: "${DOCKER_ELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
       dockerLogin(secret: "${DOCKERHUB_SECRET}", registry: 'docker.io')
     }
+    if (nodeOS().contains('darwin')) {
+      sh 'env | sort'
+    }
```
Comment (kuisathaverat) [inline Jenkinsfile:685]: Do you need it on pro?
```suggestion
```

**Review by kuisathaverat — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (v1v) [reply Jenkinsfile:685]: No, but somehow https://github.com/elastic/beats/pull/29371 works literally fine, but in this PR is not working out of the box... so I need to debug what's the reason that the `gvm` is not in the PATH.

File: Jenkinsfile [hunk 3]
```diff
@@ -989,7 +989,7 @@ def normalise(String directory) {
 */
 def dumpVariables(){
   echo "### MAGE DUMP ###"
-  cmd(label: 'Dump mage variables', script: 'mage dumpVariables')
+  cmd(label: 'Dump mage variables', script: 'mage dumpVariables || true')
```
Comment (v1v) [inline Jenkinsfile:992]:
```suggestion
  cmd(label: 'Dump mage variables', script: 'mage dumpVariables')
```

File: Jenkinsfile [hunk 4]
```diff
@@ -634,7 +634,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
 
   if(isUnix()) {
     gox_flags = (isArm() && is64arm()) ? '-arch arm' : '-arch amd64'
-    path = "${env.WORKSPACE}/bin:${env.PATH}"
+    path = "${env.WORKSPACE}/bin:${env.PATH}:/usr/local/bin"
```
Comment (v1v) [inline Jenkinsfile:637]: At the moment the orka ephemeral workers don't have the `/usr/local/bin` in their PATH environment variable

```
[2021-12-15T10:23:58.372Z] PATH=/Users/admin/workspace/PR-29426-4-fae0730c-b250-44cb-907a-866a5b5b3d7b/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

Comment (v1v) [inline Jenkinsfile:637]: At the moment the orka ephemeral workers don't have the `/usr/local/bin` in their PATH environment variable

```
[2021-12-15T10:23:58.372Z] PATH=/Users/admin/workspace/PR-29426-4-fae0730c-b250-44cb-907a-866a5b5b3d7b/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

image

**Review by cachedout — APPROVED** [review]

**Comment by v1v** [comment]
@Mergifyio backport 7.17

**Pull request merged**